.github/workflows/build.yml

```diff
@@ -158,3 +164,12 @@ jobs:
         env:
           TEST_FILE_SUB_DIR: ${{ matrix.test_dir }}
         run: sudo -E ./ci/${{ matrix.os_name }}_runner.sh script
+
+      - if: ${{ steps.cache-images.outputs.cache-hit != 'true' }}
+        name: Save docker images
+        run: |
+          # free some space to avoid disk full
+          sudo rm -rf /usr/local/lib/android /usr/share/dotnet /usr/share/swift
```

Comment:
Before that, all CIs were working fine. So I think your modification caused the lack of disk space, but your modification is obviously irrelevant to the content you submitted this time, so I cannot accept it.